### PR TITLE
3.3

### PR DESCRIPTION
**Pull request by olu144**
complete step 3 (json)

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -19,14 +19,46 @@
...
+    units.add("pounds");
     response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    response.getWriter().println("Hello Olu!");
+    String json = convertToJson(units);
+    response.setContentType("application/json;");
+    response.getWriter().println(json);
+  }
+
+  private String convertToJson(ArrayList<String> units) {
```
Comment (dborkan) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:43]: I think this code will be easier to write and maintain if you use a library to do it.  The documentation suggests using Gson - you can see an example of that here: https://github.com/googleinterns/step/blob/master/walkthroughs/week-3-server/examples/server-stats/src/main/java/com/google/sps/servlets/ServerStatsServlet.java

You will probably need to add a line like `import com.google.gson.Gson;` at the top, then you can use a function like this: https://github.com/googleinterns/step/blob/master/walkthroughs/week-3-server/examples/server-stats/src/main/java/com/google/sps/servlets/ServerStatsServlet.java#L71

**Review by dborkan — CHANGES_REQUESTED** [review]

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -19,14 +19,46 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-
+  ArrayList<String> units=new ArrayList<String>();
```
Comment (dslucas) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:27]: Can this variable be private?

Also, let's add a space before and after the = (equals sign).

**Review by dslucas — CHANGES_REQUESTED** [review]

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 1 above)
Comment (olu144) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:43]: Addressed in 3.3changes

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 2 above)
Comment (olu144) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:27]: Sure can!
Addressed in 3.3changes

**Pull request closed without merging**